### PR TITLE
[INFRA-4918] Remove All Influx/Graphite Code

### PR DESCRIPTION
**Pull request by aaronabf**
Let me know if there is any code I missed (or anything I shouldn't have removed).

### CODE REVIEW

**Comment by mahendra** [comment]
❤️

**Comment by jimjh** [comment]
delete everything!

**Comment by aaronabf** [comment]
@elainearbaugh Got the tests working. Let me know when would be the best time to roll out these changes.

File: README.md [hunk 1]
```diff
@@ -4,11 +4,10 @@ Cabot
...
 
-It provides a web interface that allows you to monitor services (e.g. "Stage Redis server", "Production ElasticSearch cluster") and send telephone, sms or hipchat/email alerts to your on-duty team if those services start misbehaving or go down - all without writing a line of code. Best of all, you can use data that you're already pushing to Graphite/statsd to generate alerts, rather than implementing and maintaining a whole new system of data collectors.
+It provides a web interface that allows you to monitor services (e.g. "Stage Redis server", "Production ElasticSearch cluster") and send telephone, sms or hipchat/email alerts to your on-duty team if those services start misbehaving or go down - all without writing a line of code. Best of all, you can use data that you're already pushing to statsd to generate alerts, rather than implementing and maintaining a whole new system of data collectors.
 
 You can alert based on:
 
-*   Metrics from [Graphite](https://github.com/graphite-project/graphite-web)
 *   Status code and response content of web endpoints
 *   [Jenkins](http://jenkins-ci.org) build statuses
```
Comment (elainearbaugh) [inline README.md]: We should update this sometime (add ES/TCP), change the intro, etc. Doesn't have to be in this PR tho

**Comment by aaronabf** [comment]
Yay! Thanks for the review. I'll deploy this to stage tomorrow.

**Pull request merged**